index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microverse week 1</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
+  </body>
+</html>
+https://github.com/Alazar721/hellomicroverse/pulls
+https://github.com/Alazar721/hellomicroverse.git

```

Comment:
- [ ] Please consider deleting these links, they don't serve any purpose and are outside the `<html>` tag which is not a good markup structure. 🤓 🛠️